tests/test_data.py

```diff
@@ -582,6 +572,48 @@ def test_tabulate_behavior_data_center_out(self):
                                                include_handdata=False, include_eyedata=False)
         self.assertEqual(len(df), 18) # should be the same df as above
 
+    def test_tabulate_kinematic_data(self):
+        subjects = [self.subject, self.subject]
+        ids = [self.te_id, self.te_id]
+        dates = [self.date, self.date]
+
+        df = tabulate_behavior_data_center_out(write_dir, subjects, ids, dates, df=None, 
+                                               include_center_target=True,
+                                               include_handdata=False, include_eyedata=False)
+
+        start_times = [t[0] for t in df['event_times']] # maybe we could update the df to have start and end times directly
```

Comment:
I think it's more flexible for the df to have all events & event_times and you specify the start and end times before calling tabulate_kinematic_data. For example in tracking task, the first and last event of each segment are not really the start and end times because I have to exclude the ramp period. If I wanted to include the ramp period in future analyses, it'd be faster to skip this step than re-generate the df with correct start & end times.